client/src/utils/uploadbox.js

```diff
@@ -66,48 +66,46 @@ function tusUpload(data, index, tusEndpoint, cnf) {
     });
 }
 
+// Posts chunked files to the API.
+export function submitUpload(config) {
+    // set options
+    const cnf = Object.assign(
+        {},
+        {
+            data: {},
+            success: () => {},
+            error: () => {},
+            warning: () => {},
+            progress: () => {},
+            attempts: 70000,
+            timeout: 5000,
+            url: null,
+            error_file: "File not provided.",
+            error_attempt: "Maximum number of attempts reached.",
+            error_tool: "Tool submission failed.",
+            chunkSize: 10485760,
+        },
+        config
+    );
```

Comment:
The more modern approach (and more readable, at least to me who hasn't seen a lot of JS) seems to be
```suggestion
    const cnf = {
        data: {},
        success: () => {},
        error: () => {},
        warning: () => {},
        progress: () => {},
        attempts: 70000,
        timeout: 5000,
        url: null,
        error_file: "File not provided.",
        error_attempt: "Maximum number of attempts reached.",
        error_tool: "Tool submission failed.",
        chunkSize: 10485760,
        ...config,
    }
```